lib/rom/mapper/attribute_dsl.rb

```diff
@@ -30,6 +29,30 @@ def initialize(attributes, options)
         @reject_keys = options.fetch(:reject_keys)
       end
 
+      # Redefine the prefix for the following attributes
+      #
+      # @example
+      #
+      #   dsl = AttributeDSL.new([])
+      #   dsl.attribute(:prefix, 'user')
+      #
+      # @api public
+      def prefix(value = nil)
+        @prefix = value
+      end
```

Comment:
hmm I left a comment under a commit and it's not visible in this PR for some reason...so I'm gonna repeat myself:

this should be, conventionally, done like that:

``` ruby
def prefix(value = Undefined)
  if value.equal?(Undefined)
    @prefix
  else
    @prefix = value
  end
end
```

we could probably have that as a support extension and simply define those methods via some macro